### DISCUSSION TITLE
Manual refresh needed?

### DISCUSSION THREAD

**Discussion by ddddami** [ngtuonghy/live-server-nvim#5] [Q&A]
Hiya @ngtuonghy . Nice project, I wonder do I have to refresh the browser for changes to reflect? Or it's an issue on my end.

**Comment by ngtuonghy** [comment]
Are you tried restarting neovim?

**Reply by ddddami** [reply]
Hi, yes. And I still have to refresh the page to see changes

**Reply by ngtuonghy** [reply]
https://github.com/user-attachments/assets/e00257c7-376b-4c2e-a98a-9c9a2f06681e

everything works
